website/docs/r/cloud_aws_provider.html.markdown

```diff
@@ -35,7 +35,7 @@ Allowed values: "no", "yes"
 Allowed values: "no", "yes"
 * `name_alias` - (Optional) name_alias for object cloud_aws_provider.
 * `provider_id` - (Optional) provider_id for object cloud_aws_provider.
-* `region` - (Optional) which AWS region to manage.
+* `region` - (Optional) which AWS region to manage. \[Supported only in Cloud APIC 4.2(x) or earlier\]
```

Comment:
Can the description for region here be the same as the one in: website/docs/d/cloud_aws_provider.html.markdown?

Reply:
Updated description for region attribute.